feat: official resources section (#967)
italia/eidas-it-wallet-docs: docs/it/functionalities.rst

    Elemento grafico “IT-Wallet ID” su sfondo chiaro  

> Le Risorse Ufficiali dell’“IT-Wallet ID” sono disponibili all’interno della Sezione :ref:`official-resources:Risorse Ufficiali`. Maggiore documentazione aggiuntiva sarà prossimamente disponibile sul sito ufficiale, come indicato nella sezione :ref:`official-resources:Risorse Ufficiali`. 

Reviewer: ```suggestion
La Risorsa Ufficiale dell’“IT-Wallet ID” è disponibile all’interno della Sezione :ref:`official-resources:Risorse Ufficiali`. Maggiore documentazione aggiuntiva sarà prossimamente disponibile sul sito ufficiale, indicato nella sezione :ref:`official-resources:Risorse Ufficiali`. 
```

- DEVE riportare la dicitura "Entra con IT-Wallet"; 

> - DOVREBBE essere sempre accompagnato da un link esterno (ad es. "Scopri di più") che rimanda al sito ufficiale del Sistema IT-Wallet, come indicato nella sezione :ref:`official-resources:Risorse Ufficiali`; 

Reviewer: ```suggestion
- DOVREBBE essere sempre accompagnato da un link esterno (ad es. "Scopri di più") che rimanda al sito ufficiale del Sistema IT-Wallet, indicato nella sezione :ref:`official-resources:Risorse Ufficiali`; 
```

> Il pulsante è disponibile in 3 varianti (default / M / L ) ed in formato "get" (chiamata ad una pagina esterna) e "post" (form interna al pulsante). I riferimenti al codice html e alla Brand Identity corretta verranno inclusi nelle versioni successive delle presenti regole tecniche.

Reviewer: ```suggestion
Il pulsante è disponibile in tre varianti di dimensione (S - default / M / L ) come da Design System .Italia, ed in formato "get" (chiamata ad una pagina esterna) e "post" (form interna al pulsante). Oltre alle varianti di dimensione, sono previste due varianti di pulsante a larghezza fissa utilizzabili in situazioni in cui è preferibile mantenere coerenza con altri pulsanti simili: 

• la versione a larghezza fissa con icona a sinistra e testo centrato; 
• la versione a larghezza fissa con icona e testo centrati. 

La Risorsa Ufficiale dell'Authentication button html è disponibile nella sezione :ref:`official-resources:Risorse Ufficiali` di queste Specifiche Tecniche. 
```

> Il pulsante è disponibile in 3 varianti (default / M / L ) ed in formato "get" (chiamata ad una pagina esterna) e "post" (form interna al pulsante). I riferimenti al codice html e alla Brand Identity corretta verranno inclusi nelle versioni successive delle presenti regole tecniche.

Reviewer: ```suggestion
Il pulsante è disponibile in tre varianti di dimensione (S - default / M / L ), come da Design System .Italia, e in formato "get" (chiamata ad una pagina esterna) e "post" (form interna al pulsante). Oltre alle varianti di dimensione, sono previste due varianti di pulsante a larghezza fissa, utilizzabili in situazioni in cui è preferibile mantenere coerenza con altri pulsanti simili: 
• la versione a larghezza fissa con icona a sinistra e testo centrato; 
• la versione a larghezza fissa con icona e testo centrati. 
La Risorsa Ufficiale dell'Authentication button html è disponibile nella sezione :ref:`official-resources:Risorse Ufficiali` di queste Specifiche Tecniche. 
```

Reviewer: The piece above should also be edited in the english version, as follows:

> The button is available in three variants of dimensions (S - default / M / L ) as per the .Italia Design System, and in "get" (call to an external page) and "post" (form inside the button) formats. In addition to the variants of dimensions, are provided two variants of button with fixed length, to be used in situations where it's preferable to keep consistency between similar buttons:
> - the fixed-width version with icon on the left and centered text;
> - the fixed‑width version with icon and text centered.
>
> The Official Resource of the Authentication Button is available in the :ref:`official-resources:Official Resources` of these Technical Specification.

Author: done 71e7e44941878c41a31d296559cc9c59eb64d8f9

**Authentication Button "Entra con IT-Wallet" - svg**

> Per approfondimenti sull'Authentication button consultare il Brand Manual, come indicato nella sezione :ref:`official-resources:Risorse Ufficiali`. La Risorsa Ufficiale dell'Authentication button è disponibile nella sezione :ref:`official-resources:Risorse Ufficiali` di queste Specifiche Tecniche. 

Reviewer: ```suggestion
Per approfondimenti sull'Authentication button consultare il Brand Manual, indicato nella sezione :ref:`official-resources:Risorse Ufficiali`. La Risorsa Ufficiale dell'Authentication button è disponibile nella sezione :ref:`official-resources:Risorse Ufficiali` di queste Specifiche Tecniche. 
```
